community/modules/compute/schedmd-slurm-gcp-v6-nodeset-tpu/main.tf

```diff
@@ -34,9 +34,8 @@ locals {
     docker_image = var.docker_image
 
     enable_public_ip = !var.disable_public_ips
-    # TODO: rename to subnetwork_self_link, requires changes to the scripts
-    subnetwork      = var.subnetwork_self_link
-    service_account = var.service_account
-    zone            = var.zone
+    subnetwork       = var.subnetwork_self_link
+    service_account  = var.service_account
+    zones            = toset(concat([var.zone], tolist(var.zones)))
```

Comment:
This is breaking change, there is no support for `tpu_nodeset.zones` in neither controller wrapper nor SlurmGCP module.

Reply:
I've created a PR to support this in the SlurmGCP module: https://github.com/GoogleCloudPlatform/slurm-gcp/pull/16